k8s/deployment.yaml

```diff
@@ -69,3 +69,5 @@ spec:
       containers:
       - name: agent-sandbox-controller
         image: ko://sigs.k8s.io/agent-sandbox/cmd/agent-sandbox-controller # placeholder value, replaced by deployment scripts
+        args:
+        - --extensions
```

Comment:
I think we can remove this because we might need to target the implementation of this flag in a different PR so that we can manage core vs extensions deployment. #109